Mono.Cecil/BaseAssemblyResolver.cs

```diff
@@ -158,8 +158,14 @@ AssemblyDefinition SearchDirectory (AssemblyNameReference name, IEnumerable<stri
 			foreach (var directory in directories) {
 				foreach (var extension in extensions) {
 					string file = Path.Combine (directory, name.Name + extension);
-					if (File.Exists (file))
-						return GetAssembly (file, parameters);
+					if (File.Exists (file)) {
```

Comment:
Could you invert the condition and continue if there's no file to reduce indentation?